### PR TITLE
docs(cli): add dev-contracts to newdocs

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: next-docs/pages/cli/dev-contracts.mdx [hunk 1]
```diff
@@ -1 +1,46 @@
...
+- Use the `--rpc <url>` command line parameter.
+- Specify the URL as the `eth_rpc_url` value [in the `local` profile in `foundry.toml`](https://book.getfoundry.sh/config/).
+- Do not specify an RPC at all, in which case `mud dev-contracts` starts an [`anvil`](https://book.getfoundry.sh/anvil/) instance for you.
+
+These are the command line options you can specify on `mud dev-contracts`:
+
+| Option                  | Meaning                                  | Type    | Default value                                                                                |
+| ----------------------- | ---------------------------------------- | ------- | -------------------------------------------------------------------------------------------- |
+| `--version`             | Show version number                      | boolean | `false`                                                                                        |
+| `--configPath`          | Path to the config file                  | string  | `foundry.toml`                                                                               |
```
Comment (alvrs) [inline next-docs/pages/cli/dev-contracts.mdx:41]:
```suggestion
| `--configPath`          | Path to the config file                  | string  | `mud.config.ts`                                                                               |
```

Comment (alvrs) [reply next-docs/pages/cli/dev-contracts.mdx:41]: missed this one in the `mud deploy` docs, we also need to update it there

**Review by alvrs — COMMENTED** [review]
we just added a `--worldAddress` param to `dev-contracts`: https://github.com/latticexyz/mud/pull/1892 - should add that to this docs page too

File: next-docs/pages/cli/dev-contracts.mdx (hunk 1 above)
Comment (qbzzt) [reply next-docs/pages/cli/dev-contracts.mdx:41]: https://github.com/latticexyz/mud/pull/1903

**Comment by qbzzt** [comment]
> we just added a `--worldAddress` param to `dev-contracts`: #1892 - should add that to this docs page too

Done.

File: next-docs/pages/cli/dev-contracts.mdx (hunk 1 above)
Comment (alvrs) [reply next-docs/pages/cli/dev-contracts.mdx:41]: Thanks for fixing it over there. We also need to fix it here though.

**Pull request merged**